.github/workflows/build_latest.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        qt-version: [ '5.12.*', '5.15.*' ]
+        qt-version: [ '5.12.*', '5.15.*', '6.5.*' ]
```

Comment:
You also have to add the `qt5compat` module in the "Install Qt" action. As it was done in the other PR:
```
with:
  version: ${{ matrix.qt-version }}
  modules: ${{startsWith(matrix.qt-version, '6') && 'qt5compat' || '' }}
  ...
```